src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute in the link tag should point to a CSS file, not an SCSS file. SCSS files need to be compiled into CSS before they can be linked in an HTML file.